JDBC数据源构建url多个/，导致无法创建连接

master分支最新版本0.11.0，JDBC数据源在构建url时会多个/，导致无法创建连接。

```
020-11-20 09:59:49 [main] [INFO ] c.b.h.l.r.j.RowFetcher [] - Connect to database /jdbc:mysql://ip:3306/enc_original_database?useSSL=false&characterEncoding=UTF-8&rewriteBatchedStatements=true&useServerPrepStmts=false&autoReconnect=true
2020-11-20 09:59:49 [main] [ERROR] c.b.h.l.u.Printer [] - Failed to load, cause: Failed to connect database via 'jdbc:mysql://ip:3306'
```

--- a/pom.xml
+++ b/pom.xml
@@ -182,7 +182,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.13</version>
+            <version>4.5.2</version>
         </dependency>
         <dependency>
             <groupId>org.tukaani</groupId>

--- a/pom.xml
+++ b/pom.xml
@@ -182,7 +182,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.10</version>
+            <version>4.5.13</version>
         </dependency>
         <dependency>
             <groupId>org.tukaani</groupId>

--- a/src/main/java/com/baidu/hugegraph/loader/constant/Constants.java
+++ b/src/main/java/com/baidu/hugegraph/loader/constant/Constants.java
@@ -33,6 +33,8 @@ public final class Constants {
     public static final String HTTPS_PREFIX = "https://";
     public static final String JSON_SUFFIX = ".json";
     public static final String GROOVY_SCHEMA = "schema";
+    public static final String TRUST_STORE_FILE = "conf/hugegraph.truststore";
+    public static final String TRUST_STORE_PASSWORD = "hugegraph";
 
     public static final String FIELD_VERSION = "version";
     public static final String V1_STRUCT_VERSION = "1.0";

--- a/src/main/java/com/baidu/hugegraph/loader/executor/LoadOptions.java
+++ b/src/main/java/com/baidu/hugegraph/loader/executor/LoadOptions.java
@@ -80,12 +80,12 @@ public final class LoadOptions {
     @Parameter(names = {"--trust-store-file"}, arity = 1,
                description = "The path of client truststore file used when https " +
                              "protocol is enabled")
-    public String trustStoreFile = "conf/hugegraph.truststore";
+    public String trustStoreFile = null;
 
     @Parameter(names = {"--trust-store-password"}, arity = 1,
                description = "The password of the client truststore file used " +
                              "when https protocol is enabled")
-    public String trustStorePassword = "hugegraph";
+    public String trustStorePassword = null;
 
     @Parameter(names = {"--token"}, arity = 1,
                description = "The token of graph for authentication")

--- a/src/main/java/com/baidu/hugegraph/loader/util/HugeClientHolder.java
+++ b/src/main/java/com/baidu/hugegraph/loader/util/HugeClientHolder.java
@@ -56,13 +56,21 @@ public final class HugeClientHolder {
                                 .configPool(options.maxConnections,
                                             options.maxConnectionsPerRoute);
             if (useHttps) {
-                String homePath = System.getProperty("loader.home.path");
-                E.checkArgument(StringUtils.isNotEmpty(homePath),
-                                "The system property 'loader.home.path' " +
-                                "can't be empty when enable https protocol");
-                String trustFile = Paths.get(homePath, options.trustStoreFile)
-                                        .toString();
-                builder.configSSL(trustFile, options.trustStorePassword);
+                String trustFile;
+                if (options.trustStoreFile == null) {
+                    String homePath = System.getProperty("loader.home.path");
+                    E.checkArgument(StringUtils.isNotEmpty(homePath),
+                                    "The system property 'loader.home.path' " +
+                                    "can't be empty when enable https protocol");
+                    trustFile = Paths.get(homePath, Constants.TRUST_STORE_FILE)
+                                     .toString();
+                } else {
+                    trustFile = options.trustStoreFile;
+                }
+                String password = options.trustStorePassword == null ?
+                                  Constants.TRUST_STORE_PASSWORD :
+                                  options.trustStorePassword;
+                builder.configSSL(trustFile, password);
             }
             return builder.build();
         } catch (IllegalStateException e) {

